### PR TITLE
istioctl/cmd/add-to-mesh:optimize code

### PR DESCRIPTION
**Pull request by tanjunchen**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[x] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by morvencao — APPROVED** [review]

**Review by morvencao — COMMENTED** [review]
/retest

File: istioctl/cmd/add-to-mesh.go [hunk 1]
```diff
@@ -230,6 +228,19 @@ See also 'istioctl experimental remove-from-mesh service' which does the reverse
 	return cmd
 }
 
+func injectSideCarIntoDeployments(client kubernetes.Interface, deps []appsv1.Deployment, sidecarTemplate, valuesConfig,
```
Comment (irisdingbj) [inline istioctl/cmd/add-to-mesh.go:231]: This might be a better naming, but no optimization for the code path if I understand correctly.

**Review by shamsher31 — APPROVED** [review]

File: istioctl/cmd/add-to-mesh.go (hunk 1 above)
Comment (tanjunchen) [reply istioctl/cmd/add-to-mesh.go:231]: Basically yes

**Pull request merged**